terraform/cloudwatch_alarm_sns_topic.tf

```diff
@@ -0,0 +1,15 @@
+# ------------------------------------------------------------------------------
+# Create the SNS topic that allows email to be sent for CloudWatch
+# alarms.  Subscribe the account email to the new SNS topic.
+# ------------------------------------------------------------------------------
+
+resource "aws_sns_topic" "cloudwatch_alarm" {
+  name         = "cloudwatch-alarms"
+  display_name = "cloudwatch_alarms"
+}
+
+resource "aws_sns_topic_subscription" "account_email" {
+  endpoint  = "cisa-cool-group+cyhy@trio.dhs.gov"
```

Comment:
In the COOL I just pull the email associated with the account from the AWS Organization via the master account.  Is there a way to do something similar here?

Reply:
It's the email we want to send alerts to for production workspaces, but I don't think we want to clutter that email with testing deployment updates in non-production workspaces.

Reply:
Right, I get that, but that still doesn't answer my question:  _In the COOL I just pull the email associated with the account from the AWS Organization via the master account. Is there a way to do something similar here?_

Reply:
I guess my suggestion was basically variable with a default to override? I was thinking of it along the lines of the AMI prefix variable I added. I'm not aware offhand of a way to do it the way you describe in this account since it's unfortunately not structured like COOL accounts.

Reply:
Please see commit 62c5cee.